src/subcommand/server.rs

```diff
@@ -1629,10 +1628,11 @@ mod tests {
     test_server.mine_blocks(101);
 
     test_server.assert_response_regex(
-    "/",
-    StatusCode::OK,
-    ".*<ol start=101 reversed class=blocks>\n(  <li><a href=/block/[[:xdigit:]]{64}>[[:xdigit:]]{64}</a></li>\n){100}</ol>.*"
-  );
+      "/",
+      StatusCode::OK,
+      // ".*<ol start=101 reversed class=blocks>\n(    <li><a href=/block/[[:xdigit:]]{64}>[[:xdigit:]]{64}</a></li>\n){100}</ol>.*"
+      ".*<ol start=101 reversed class=blocks>.*</ol>.*",
```

Comment:
Hmm, yeah, these tests seem a bit frustrating to maintain. Still, this is probably not a change we want to make:

- It looks like the test is supposed to test that there's a limit to the number of shown blocks. The test doesn't test that anymore. If we don't want to test that anymore we should remove the test. If we do, we need to fix it.
- We probably don't want old regexes in comments like this.